httpx/config.py

```diff
@@ -72,27 +75,35 @@ def with_overrides(
             return self
         return SSLConfig(cert=cert, verify=verify)
 
-    def load_ssl_context(self) -> ssl.SSLContext:
+    def load_ssl_context(
```

Comment:
What are your thoughts on making http_versions a part of the SSLConfig object instead of being piped through?

Would also include updating `__eq__` and hash too.

Reply:
I don't know. That gets a bit different once we're approaching HTTP/3, or if we ever decided to support HTTP/2 upgrades over http. My feeling is that it's a "possibly", and that we should treat it as a later refactoring if we did decide to do so.